include/azure_kinect_ros_driver/k4a_exposure_calibration.h

```diff
@@ -0,0 +1,158 @@
+     *          blue value provided by the user in k4aAutoTuneExposure
+     * @param[in] target_blue_val target blue value originally requested in k4aAutoTuneExposure
+     * @param[in] current_average_blue_value the average blue value of the image at a point in time
+     * @param[out] error_code error code included in response
+     * @param[out] res_msg human-readable error message included in response
+     */
+    bool k4aTargetBlueCheck(int target_blue_val, int current_avg_blue_value, int& error_code, std::string& res_msg);
+    
+    /**
+     * @brief check if target_blue_value is in appropriate range
+     * @param[in] target_blue_value target blue value originally requested in k4aAutoTuneExposure
+     * @param[out] error_code error code included in response
+     * @param[out] res_msg human-readable error message included in response
+     */
+    bool k4aBlueBoundsCheck(int target_blue_value, int& error_code, std::string& res_msg);
+    
+    /**
+     * @brief did the node receive an image at all?
+     * @param[in] mat current OpenCV mat stored in latest_k4a_image for k4aAutoTuneExposure loop
+     * @param[out] error_code error code included in response
+     * @param[out] res_msg human-readable error message included in response
+     */
+    bool k4aImagePopulatedCheck(cv::Mat& mat, int& error_code, std::string& res_msg);
+
+private:
+    /**
+     * @brief callback for /rgb/raw/image subscription
+     * @details This callback handles updating the current image for the node.
+     *          It converts the ROS image message to an OpenCV mat for auto tuning calculations.
+     * @param[out] msg message received from image transport subscriber subscribed to /rgb/raw/image
+     */
+    void rgbRawImageCallback(const sensor_msgs::ImageConstPtr& msg);
+
+    /**
+     * @brief callback for k4aUpdateExposure
+     * @details This callback handles requests to update the camera exposure from the
+     *          k4a_update_exposure service.
+     * @param[in] req request received from calling k4a_update_exposure
+     * @param[out] res response sent from k4a_update_exposure
+     */
+    bool k4aUpdateExposureCallback(azure_kinect_ros_driver::k4a_update_exposure::Request &req,
+                                   azure_kinect_ros_driver::k4a_update_exposure::Response &res);
+    
+    /**
+     * @brief callback for k4aAutoTuneExposure
+     * @details This callback handles requests to update the camera exposure from the
+     *          k4a_auto_tune_exposure service.
+     * @param[in] req request received from calling k4a_auto_tune_exposure
+     * @param[out] res response sent from k4a_auto_tune_exposure
+     */
+    bool k4aAutoTuneExposureCallback(azure_kinect_ros_driver::k4a_auto_tune_exposure::Request &req,
+                                     azure_kinect_ros_driver::k4a_auto_tune_exposure::Response &res);
+
+    // private members
+    ros::NodeHandle nh_;
+    image_transport::Subscriber subRGBRaw;
```

Comment:
add an underscore suffix for all private members (like nh_). If you're going to be consistent. Alternatively I enjoy the m_ prefix so like m_nh or m_subRGBRaw. Typically folks who use the underscore (_) tend to be old C programmers and the m_ are C++ programmers or mSubRGBRaw are java programmers. 

I personally don't care as long as you're consistent with your vars.